lib/symengine/matrix_base.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class MatrixBase
+    def inspect
+      "#<#{self.class}()>"
```

Comment:
OK. Let's just not forget to recheck (and possibly join) the code when `SparseMatrix` will come to live.